stdlib/src/collections/list.mojo

```diff
@@ -864,3 +874,18 @@ struct List[T: CollectionElement](
 
 fn _clip(value: Int, start: Int, end: Int) -> Int:
     return max(start, min(value, end))
+
+
+fn _move_pointee_into_many_elements[
+    T: CollectionElement, //, hint_trivial_type: Bool
+](dest: UnsafePointer[T], src: UnsafePointer[T], size: Int,):
+    @parameter
+    if hint_trivial_type:
+        memcpy(
+            dest=dest.bitcast[Int8](),
```

Comment:
`memcpy` requires `Int8` (otherwise the compilation fails) but since we wanted to standardize on `UInt8` we may want to use `UInt8` in the `memcpy` signature in the future.

Reply:
Ah, makes sense.  +1 to changing `memcpy` to using `UInt8` in a future PR.